openforcefield/topology/topology.py

```diff
@@ -1616,7 +1622,8 @@ def from_parmed(parmed_structure, unique_molecules=None):
         # TODO: Implement functionality
         raise NotImplementedError
 
-    def to_parmed(self):
+    # TODO: Jeff prepended an underscore on this before 0.2.0 release to remove it from the API. Fix?
```

Comment:
Why?

Reply:
After some thought, this one is also probably safe to expose. I've removed the underscore and added

```
        .. warning:: This functionality will be implemented in a future toolkit release.
```
to the docstring